AEXS/aex-1.md

```diff
@@ -6,7 +6,7 @@ Title: AEX process
 Author: Sascha Hanse <shanse@posteo.de>
 License: CC0-1.0
 Discussions-To: https://forum.aeternity.com/t/aeternity-best-current-practices-aexpansions/2461
-Status: Active
+Status: Review
```

Comment:
The status of AEX-1 should not change

Reply:
Right, probably some error while reading it.